tensorflow/lite/micro/kernels/conv.h

```diff
@@ -72,6 +72,16 @@ TfLiteStatus CalculateOpDataConv(TfLiteContext* context, TfLiteNode* node,
 
 TfLiteStatus ConvPrepare(TfLiteContext* context, TfLiteNode* node);
 
+#if defined(XTENSA)
+// Returns a TfLiteRegistration struct for kernel variant that only supports
+// int8 inputs and outputs.
+TfLiteRegistration Register_CONV_2D_INT8REF();
+#else 
+inline TfLiteRegistration Register_CONV_2D_INT8REF() {
+  return Register_CONV_2D();
```

Comment:
`Register_CONV_2D` declaration needs to be moved from micro_ops.h to the current header.

Reply:
Good call. Added conv.h to micro_mutable_op_resolver.h as well.